app/src/main/java/com/github/gogetters/letsgo/activities/ChatActivity.kt

```diff
@@ -1,54 +1,151 @@
 package com.github.gogetters.letsgo.activities
 
 import android.os.Bundle
+import android.util.Log
 import android.view.View
 import android.widget.EditText
 import android.widget.ListView
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import com.github.gogetters.letsgo.R
 import com.github.gogetters.letsgo.chat.ChatMessage
 import com.github.gogetters.letsgo.chat.MessageAdapter
+import com.github.gogetters.letsgo.database.Database
+import com.github.gogetters.letsgo.database.types.MessageData
 import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.database.ChildEventListener
+import com.google.firebase.database.ktx.getValue
 import java.util.*
 
 
 class ChatActivity : AppCompatActivity() {
-    // the text field to which messages are written
+
+    private val _defaultUsername = "Default User"
```

Comment:
does this need an underscore? Kotlin naming conventions says it should be reserved for private "copies" of public variables. Here there is no public "defaultUsername" so might as well make it without the underscore no?